<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Terminal</class>
 <widget class="QDialog" name="Terminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminal GPS to UDP</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="pte_read">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cb_rewind">
       <property name="text">
        <string>Auto scroll</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_warp">
       <property name="text">
        <string>Warp Lines</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_autoDelete">
       <property name="text">
        <string>Auto delete</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Lines count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_lineCount">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_clear_read">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_startStop">
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Extract --&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_extract">
       <property name="text">
        <string>$GPGGA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>change to --&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_change">
       <property name="text">
        <string>$GNGGA</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>UDP send:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_udpSend"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_port">
       <property name="text">
        <string>8477</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_send">
       <property name="text">
        <string>Start send</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
